rules/linux/env_binary_shell_evasion.toml

```
@@ -0,0 +1,25 @@
+[metadata]
+creation_date = "2022/02/24"
+maturity = "production"
+updated_date = "2022/02/24"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies Unix binary env absue to break out from restricted environments by spawning an interactive system shell"
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux Restricted Shell Breakout via env Shell Evasion"
+references = ["https://gtfobins.github.io/gtfobins/env/"]
+risk_score = 47
+rule_id = "72d33577-f155-457d-aad3-379f9b750c97"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "GTFOBins"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where process.name : "env" and process.args : "/bin/sh" and event.type : "start"
```

For linux systems, we can tighten these queries down with case sensitivity by using `==` instead of `:`. And for values set by the agents, such as `event.type` we should be safe to always use `==`

```suggestion
process where event.type == "start" and process.name == "env" and process.args == "/bin/sh"
```

---

Do we have a sense of noise for this one? Any way to tighten it down?

---

```suggestion
process where process.name : "env" and process.args_count == 2 and process.args : ("/bin/sh", "/bin/bash") and event.type == "start"
```